### PR TITLE
Pepper spray module for sec modsuits, Atmos resin module and nukie pyro bundle flamethrower made into module

### PR DESCRIPTION
**Pull request by CursedBirb**
## About The Pull Request
Pyro bundle was broken as when you brought it you would get modsuit and pyro backpack forcing you to either use armor or use backpack without fire protection(which modsuit has). Thanks to Fikou work it became a module
Anti-Tider on the other hand is rarely used(i have never seen it used when i was playing sec or observing) and needed a little buff so now it will also became a module while i will make normal Anti-Tider stream wider(still working on it).
Fikou is still working on code but there is a sneak peak of better pyro module
https://user-images.githubusercontent.com/47064429/154069094-da43ba6f-8666-448f-bb89-adf17df2d0e7.mp4
https://streamable.com/pmywk3 - better quality

## Changelog

:cl:
balance: Nanotrasen upgrades Anti-Tider back pack and find a way to make a compact version for security modsuit, researchers still high on caffeine creates module version of atmos resin backpack
fix: Syndicate quickly steals designs and applies them to their pyro backpack transforming it into module
names:Fikou for almost entire code work, TGMC for sound and Cursed Birb
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
Seeing an officer remove their entire backpack for a glorified pepperspray is hilarious, even moreso when it actually works. It has a certain charm to being a backpack-slot item and tbh there are cooler ways to balance it than letting everyone put it in their suit slot

**Comment by Amrabol** [comment]
> Seeing an officer remove their entire backpack for a glorified pepperspray is hilarious, even moreso when it actually works. It has a certain charm to being a backpack-slot item and tbh there are cooler ways to balance it than letting everyone put it in their suit slot

Thats why its only allowed on 4 clothes. Im still thinking that swat was overkill in that regard but it forces them out of basic sec armor. So they have to sacrifice something to be able to wear it on suit slot

**Review by tralezab — CHANGES_REQUESTED** [review]
Just make them modsuit modules

**Comment by tralezab** [comment]
And to specify: Pyro bundle is a default module on the included modsuit, anti tider backpack should still exist but also be a security modsuit module, your choice if default or research (but respect fikou if he decides on this)

**Comment by ATH1909** [comment]
> Just make them modsuit modules

aw, but then I can't do ghetto chemistry with them

File: code/modules/mod/mod_types.dm [hunk 1]
```diff
@@ -175,6 +175,17 @@
...
 
+/obj/item/mod/control/pre_equipped/elite/flamethrower
+	initial_modules = list(
+		/obj/item/mod/module/storage/syndicate,
+		/obj/item/mod/module/emp_shield,
+		/obj/item/mod/module/thermal_regulator,
+		/obj/item/mod/module/jetpack/advanced,
+		/obj/item/mod/module/flashlight,
+		/obj/item/mod/module/holster,
+		/obj/item/mod/module/flamethrower
```
Comment (Fikou) [inline code/modules/mod/mod_types.dm:186]:
```suggestion
		/obj/item/mod/module/flamethrower,
```

File: code/modules/mod/mod_theme.dm [hunk 2]
```diff
@@ -557,6 +557,7 @@
 	complexity_max = DEFAULT_MAX_COMPLEXITY - 5
 	slowdown_inactive = 0.75
 	slowdown_active = 0.25
+	inbuilt_modules = list(/obj/item/mod/module/mister/security)
```
Comment (Fikou) [inline code/modules/mod/mod_theme.dm:560]: dont make this inbuilt, make it initial in the mod object

File: code/modules/research/designs/mechfabricator_designs.dm [hunk 3]
```diff
@@ -1342,6 +1342,13 @@
 	materials = list(/datum/material/iron = 750, /datum/material/glass = 500)
 	build_path = /obj/item/mod/module/signlang_radio
 
+/datum/design/module/mod_mister/atmos
```
Comment (Fikou) [inline code/modules/research/designs/mechfabricator_designs.dm:1345]: you dont have a mod_mister path, just make this module/mister_atmos

File: code/modules/mod/mod_theme.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/mod/mod_theme.dm:560]: same with the other

File: code/modules/research/techweb/all_nodes.dm [hunk 4]
```diff
@@ -1470,6 +1470,7 @@
 		"mod_magboot",
 		"mod_tether",
 		"mod_constructor",
+		"mod_mister_atmos",
```
[the remark this answers is not in the document]
Comment (Amrabol) [reply code/modules/research/techweb/all_nodes.dm:1473]: Will make it cost also 1000 silver, 1500 titanium and 500 uranium

**Comment by Amrabol** [comment]
Will make it ready to review once i make it that Anti-Tider shoots at 3 tiles at once

**Comment by tralezab** [comment]
Thanks for putting in the work

**Review by Twaticus — APPROVED** [review]

**Comment by Amrabol** [comment]
I tried to make it that Anti-Tider shoots at 3 tiles at once, even found code that i could use for this. Yet i lack knowledge how to make that code work for Anti-Tider so for now i will just give Anti-Tider more volume(double of modsuit module(module has 500)). Will try to come back in future once i will make that code works

**Comment by Amrabol** [comment]
I wanted to make it that every pepper spray item had its purpose:
-Pepper spray - hand hold, small, little range;
-Pepper spray mister module - works in modsuit, has bigger range and capacity than hand held one, can't be stolen from hand;
-Anti-Tider backpack - superior volume, superior range, can't be stolen from hand, slows user a little(was also supposed to shoot at 3 tile lines at once but it is currently beyond my capabilites(im writing this even when its fikou who did almost all the work(i wanted to do that myself but will try to comeback to it later))).

**Pull request merged**